source/core/src/main/com/csse3200/game/components/mainmenu/MainMenuActions.java

```diff
@@ -51,6 +51,7 @@ private void onLoad() {
     private void onExit() {
         logger.info("Exit game");
         game.exit();
+        System.exit(-1);
```

Comment:
Why is the exit status -1? I'm pretty sure -1 means exited with error. Also the `game.exit()` should take care of this anyway